curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-advanced-array-methods-by-building-a-statistics-calculator/63507bcbfe4ede356e624395.md

```diff
@@ -33,6 +33,12 @@ Your `calculate` function should assign the result of the `.split()` method to t
 assert.match(code.toString(),  /array\s*=\s*value\.split()/);
 ```
 
+The `.split()` method should use a regex to split the `value` string by commas followed by any number of spaces. Use the `/,\s*/g` regex.
```

Comment:
For hint text, we should be consistent with the language and start with `You should`. This can also be simplified to the following. We should also mention the seperator just like [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split) does 

```suggestion
You should use `/,\s*/g` for the `split()` method's separator.
```